package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-sdk",
-  "version": "7.10.5",
+  "version": "7.10.6",
```

Comment:
This will be updated by our internal release process (same on package-lock updates) 😉 
```suggestion
  "version": "7.10.5",
```

Reply:
Haven't changed, it's aschen commit, i'll try to squash

Reply:
@alexandrebouthinon @Aschen I was able to fix the "commit" from aschen by switching target to `master` then back to `7-dev`.

FYI : 
- https://stackoverflow.com/a/26986320/5320409
- https://stackoverflow.com/a/45286781/5320409